pkg/controllers/monitoring/monitoring-stack/controller.go

```diff
@@ -42,7 +42,7 @@ import (
 	monv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
 )
 
-type reconciler struct {
+type msController struct {
```

Comment:
Since this isn't a controller - could we rename it back to `reconciler` ? 

see : NewControllerManagedBy which returns a `Builder` whose build method returns a `controller.Controller`

Reply:
The motivation was to not have several `reconciler` names that do different things. But I agree, `*Controller` is also not great.

Maybe the component that is added here would be better named a `ResourceManager`? That would be consistent with https://kubernetes.io/docs/concepts/cluster-administration/manage-deployment/ at least?

Reply:
resourceManager sounds good to me

Reply:
`Patcher | Updater` ? has a `Patch|Update`